apps/src/templates/sectionAssessments/MatchAssessmentsOverviewTable.jsx

```diff
@@ -0,0 +1,241 @@
+import orderBy from 'lodash/orderBy';
+import MultipleChoiceAnswerCell from './MultipleChoiceAnswerCell';
+import styleConstants from '@cdo/apps/styleConstants';
+import {setQuestionIndex} from './sectionAssessmentsRedux';
+import ReactTooltip from 'react-tooltip';
+import {optionDataPropTypeMatch} from './assessmentDataShapes';
+
+export const COLUMNS = {
+  OPTION: 0
+};
+
+const ANSWER_COLUMN_WIDTH = 100;
+const PADDING = 20;
+
+const styles = {
+  answerColumnHeader: {
+    width: ANSWER_COLUMN_WIDTH,
+    textAlign: 'center'
+  },
+  answerColumnCell: {
+    width: ANSWER_COLUMN_WIDTH,
+    padding: 0,
+    height: 40
+  },
+  notAnsweredCell: {
+    padding: 0,
+    height: 40
+  },
+  questionCell: {
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap'
+  }
+};
+
+const NOT_ANSWERED = 'notAnswered';
+
+const answerColumnsFormatter = (
+  percentAnswered,
+  {rowData, columnIndex, rowIndex, property}
+) => {
+  let percentValue = 0;
+  const answerResults = rowData.answers[columnIndex - 1] || {};
+
+  if (property === NOT_ANSWERED) {
+    percentValue = Math.round(
+      (rowData.notAnswered / rowData.totalAnswered) * 100
+    );
+  } else {
+    percentValue = Math.round(
+      (answerResults.numAnswered / rowData.totalAnswered) * 100
+    );
+  }
+
+  return (
+    <MultipleChoiceAnswerCell
+      id={rowData.id}
+      percentValue={percentValue}
+      isCorrectAnswer={!!answerResults.isCorrect}
+    />
+  );
+};
+
+class MatchAssessmentsOverviewTable extends Component {
+  static propTypes = {
+    questionAnswerData: PropTypes.arrayOf(optionDataPropTypeMatch),
+    setQuestionIndex: PropTypes.func.isRequired
+  };
+
+  state = {
+    [COLUMNS.OPTION]: {
+      direction: 'desc',
+      position: 0
+    }
+  };
+
+  getSortingColumns = () => {
+    return this.state.sortingColumns || {};
+  };
+
+  onSort = selectedColumn => {
+    this.setState({
+      sortingColumns: sort.byColumn({
+        sortingColumns: this.state.sortingColumns,
+        // Custom sortingOrder removes 'no-sort' from the cycle
+        sortingOrder: {
+          FIRST: 'asc',
+          asc: 'desc',
+          desc: 'asc'
+        },
+        selectedColumn
+      })
+    });
+  };
+
+  optionFormatter = (option, {rowData, columnIndex, rowIndex, property}) => {
+    return <div>{`${option}`}</div>;
+  };
+
+  getNotAnsweredColumn = () => ({
+    property: NOT_ANSWERED,
+    header: {
+      label: i18n.notAnswered(),
+      props: {
+        style: {
+          ...tableLayoutStyles.headerCell,
+          ...styles.answerColumnHeader
+        }
+      }
+    },
+    cell: {
+      format: answerColumnsFormatter,
+      props: {
+        style: {
+          ...tableLayoutStyles.cell,
+          ...styles.notAnsweredCell
+        }
+      }
+    }
+  });
+
+  getAnswerColumn = columnLabel => ({
+    property: 'percentAnswered',
+    header: {
+      label:
+        columnLabel.length < 30
```

Comment:
This 30 seems like a magic number. It's just based on what fits, right? Not blocking, but I wonder if there is a way to tie it to column width?